recipe/meta.yaml

```diff
@@ -9,7 +9,8 @@ source:
   sha256: faa529167b22412ea51ea44f20728a602415734d2229721d1194cea47cb730fa
 
 build:
-  number: 0
+  noarch: python
```

Comment:
You need to add the `entry_point` manually otherwise that won't work properly on Windows.

Reply:
How do I do that?

Reply:
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
I added the same entry_point as the pip package.

https://github.com/jupyter/qtconsole/blob/master/setup.py#L103